frontend/package.json

```diff
@@ -8,9 +8,12 @@
     "dist-trees": "ng build --configuration=trees",
     "dist-prod": "ng build --configuration=production",
     "docs": "typedoc --out ./src/assets/typedoc --exclude '**/*.spec.ts' ./src/ --module commonjs",
-    "e2e:cidkr": "ng e2e --port=4200 --protractor-config ./development-configurations/protractor.conf.js --suite=docker-smoke-test",
-    "e2e:xmas": "ng e2e --port=4200 --protractor-config ./development-configurations/protractor.conf.js --suite=app-xmas",
-    "e2e": "ng e2e --protractor-config ./development-configurations/protractor.conf --port=4200",
+    "e2e:cidkr": "HEADLESS=true ng e2e --port=4200 --protractor-config ./development-configurations/protractor.conf.js --suite=docker-smoke-test",
```

Comment:
Run e2e tests in headless mode locally by default. Provide additional commands to run headful if desired.